docs/storage.rst

```diff
@@ -146,210 +146,250 @@ start to diminish.
+                 - all
+               * - User
+                 - The user to which the rule applies. The reserved username of *all* may be used to apply the rule to all users.
+                 - all
+               * - Address
+                 - The network mask to allow connections from. The example shown allows connections from all IPv4 addresses. Replace this with a specific network mask for better security (e.g. `192.168.1.0/24` limits connections to just those from the network 192.168.1.xxx). Either IPV4 or IPV6 network masks may be supplied.
+                 - 0.0.0.0/0
+               * - Authentication Method
+                 - The authentication method to use. An authentication method of *trust* disables password authentication. The use of *trust* is convenient for testing, but is **not recommended for production use**. Use `md5` or `scram-sha-256` for secure authentication in production.
+                 - trust
 
-    sudo yum install -y postgresql13 postgresql13-server
+        An example entry for use in a production environment, might be as follows:
 
-Before using the PostgreSQL server, you need to first initialize the database service using the command
+        .. code-block:: ini
 
-.. code-block:: console
+           host    all    all    0.0.0.0/0    md5
 
-    sudo /usr/pgsql-13/bin/postgresql-13-setup initdb
+        .. note::
 
-You can then proceed to start the database server as follows
+           Security may be further enhanced if SSL encryption is enforced on connections. This is done by specifying a connection type of *hostssl* rather than *host* in the configuration record. You must also enable SSL support in PostgreSQL, refer to the PostgreSQL documentation for settings required for this.
 
-.. code-block:: console
+      - Save the file and exit the editor.
 
-    sudo systemctl enable --now postgresql-13
 
-Confirm if the just started service above is running by checking its status using the command
+   #. Restart the PostgreSQL Service
 
-.. code-block:: console
+      After making changes to the configuration files, you need to restart the PostgreSQL service to apply the updates.
 
-    sudo systemctl status postgresql-13
+      .. note::
 
-Next, you must create a PostgreSQL user that matches your Linux user.
+         Once the configuration is complete, ensure that the machine's firewall settings allow incoming connections to PostgreSQL's default port (5432).
 
-.. code-block:: console
+   #. Setting Up PostgreSQL Client on the Local Machine
+
+      This section explains how to set up the PostgreSQL client on the local machine (the machine where Fledge executes).
+
+      - Install PostgreSQL Client
+
+         The PostgreSQL client tools are required to allow Fledge to interact with a remote PostgreSQL server. Install the `postgresql-client` package using the package manager for your system.
```

Comment:
The PostgreSQL client tools are necessary for Fledge.....

 Install the `postgresql-client` package using your system's package manager.